src/lib/header-bidding/prebid/prebid.ts

```diff
@@ -162,35 +157,47 @@ type EnableAnalyticsConfig = {
 	};
 };
 
+type BidResponse = {
+	bidder: BidderCode;
+	width: number;
+	height: number;
+	pbCg: string;
+	cpm: number;
+	adId: string;
+	source: string;
+	mediaType: string;
+	size: string;
+	bidderCode: string;
+	[x: string]: unknown;
+};
+
 // bidResponse expected types. Check with advertisers
 type XaxisBidResponse = {
-	appnexus: {
-		buyerMemberId: string;
+	appnexus?: {
+		buyerMemberId?: string;
```

Comment:
Are these properties now optional?

Reply:
They were only used wrapped with `Partial`, this is more explicit I think